libraries/lib-time-and-pitch/StaffPadTimeAndPitch.cpp

```diff
@@ -19,38 +20,72 @@ void GetOffsetBuffer(
       offsetBuffer[i] = buffer[i] + offset;
 }
 
+int GetFftSize(int sampleRate, bool formantPreservationOn)
+{
+   if (const auto fftSize = FormantShifterLogger::GetFftSizeOverride())
```

Comment:
You use `FormantShifterLogger` directly here. This happens because `GetFftSizeOverride` must not be a part of it.

I think that you should have an empty logger for production and use FormantShifterLogger only for tests. 

Configurability must not be a part of logger.